minigotchi-ESP32/display.h

```diff
@@ -1,34 +1,38 @@
 #ifndef DISPLAY_H
 #define DISPLAY_H
 
-#include "config.h"
-#include <Adafruit_GFX.h>
 #include <Adafruit_SSD1306.h>
+#include <Adafruit_GFX.h>
 #include <SPI.h>
-#include <TFT_eSPI.h> // Defines the TFT_eSPI library for CYD
 #include <Wire.h>
+#include <TFT_eSPI.h>  // Defines the TFT_eSPI library
+#include "config.h"
 
 // SSD1306 screen
 #define SSD1306_SCREEN_WIDTH 128
 #define SSD1306_SCREEN_HEIGHT 64
 
 // Init screen
-#define SSD1306_OLED_RESET -1
+#define SSD1306_OLED_RESET -1 
 #define WEMOS_OLED_SHIELD_OLED_RESET 0 // GPIO0
 
 /** developer note:
- *
+ * 
  * the TFT_eSPI library may not require this, but these will be here regardless
- *
- */
+ * 
+*/
 
 // Define display parameters for CYD
 #define CYD_SCREEN_WIDTH 240
 #define CYD_SCREEN_HEIGHT 320
 
-// lilygo t-display-t3
-#define T_DISPLAY_S3_WIDTH 320
-#define T_DISPLAY_S3_HEIGHT 170
+// Define display parameters for TTGO_T_DISPLAY
+#define TTGO_T_DISPLAY_WIDTH 135
```

Comment:
Do we have different displays? I got the information for the display size and stuff from the product page, but if they're different I think we should make them separate values